src/Qossmic/DataTransformer/ValueObjectTransformer.php

```diff
@@ -46,7 +46,7 @@ public function __construct(ExceptionHandlerRegistry $exceptionHandlerRegistry,
      *
      * @return array<string,bool|int|string|null>|bool|int|string|null
      */
-    public function transform($value)
+    public function transform(mixed $value): array|bool|int|string|null
```

Comment:
@xabbuh this is too narrow, it must be mixed.

Reply:
Can you please open an issue with an example that would fail with the current code?

Reply:
done! https://github.com/qossmic/rich-model-forms-bundle/issues/154